test/e2e/tests/custom-token-add-approve.spec.js

```diff
@@ -49,7 +62,7 @@ describe.skip('Create token, approve token and approve token without gas', funct
           await driver.clickElement({ text: 'Save', tag: 'button' });
```

Comment:
same here, we can remove the above line of `waitForSelector`, as the below clickElement method will make sure to `findClickableElement` first

Reply:
Done